pkg/search/proxy/store/util.go

```diff
@@ -41,10 +50,27 @@ func newMultiClusterResourceVersionFromString(s string) *multiClusterResourceVer
 		return m
 	}
 	// if invalid, ignore the version
-	_ = json.Unmarshal(decoded, &m.rvs)
+	_ = unmarshalRvs(decoded, &m.rvs)
 	return m
 }
 
+func unmarshalRvs(bytes []byte, c *map[string]string) error {
+	pairs := strings.Split(string(bytes), ";")
+	result := make(map[string]string, len(pairs))
+	for _, pair := range pairs {
+		cv := strings.Split(pair, ":")
```

Comment:
As above you say, rv may be `cluster1:`, cv will be [cluster1]

Reply:
No, it will be ["cluster1", ""]